Iceberg-UI.package/IceCreateRepositoryModel.class/instance/initializePresenter.st

```diff
@@ -1,3 +1,7 @@
 initialization
 initializePresenter
-	createButton action: [ self createRepository ]
\ No newline at end of file
+	createButton action: [ self createRepository ].
+	localDirectoryLocation locationInput
+		whenTextChanged: [ :newText | 
+			self location isNotNil & subdirectory text isEmpty
+				ifTrue: [ subdirectory text: (self guessSubdirectoryIn: self location) ] ]
```

Comment:
some remarks: 
- I do not like logic so complex in an event block, I'd prefer to extract a method there.
- this: `self location isNotNil & subdirectory text isEmpty`
is in general incorrect (unnecessary). It should be better like thos: 
`self location notNil and: [ subdirectory text isEmpty ]`